src/providers/ChatAppProvider.tsx

```diff
@@ -447,6 +448,26 @@ const ChatAppContextProvider: React.FC<ChatAppContextProviderProps> = ({
 
       discardMessageDraft();
     }
+    if(file){
```

Comment:
highly recommend you install your IDE plugin for https://prettier.io/. Not having to worry about formatting is sweet :).